snuba/datasets/cdc/cdcprocessors.py

```diff
@@ -1,13 +1,28 @@
 from __future__ import annotations
+import re
 
 from abc import ABC, abstractmethod
+from datetime import datetime
 from typing import Any, Mapping, Optional, Sequence, Type
 
 from snuba.processor import MessageProcessor
 from snuba.writer import WriterTableRow
 
 KAFKA_ONLY_PARTITION = 0  # CDC only works with single partition topics. So partition must be 0
 
+POSTGRES_DATE_FORMAT_WITH_NS = "%Y-%m-%d %H:%M:%S.%f%z"
+POSTGRES_DATE_FORMAT_WITHOUT_NS = "%Y-%m-%d %H:%M:%S%z"
+
+date_with_nanosec = re.compile("^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\.")
+
+
+def parse_poostgres_datetime(date: str) -> datetime:
+    date = f"{date}00"
```

Comment:
Comment here describing the significance of this value would be helpful to have.